LED_Marquee.ino

```diff
@@ -1031,34 +1036,38 @@ void marquee() {
         uint rampup = GAMMA(JAB_MIN_BRIGHTNESS + step);
         uint rampdown = GAMMA(JAB_MIN_BRIGHTNESS + (JAB_STEPS - step));
 
-        uint r, g, b;
+        uint r0, g0, b0, r, g, b;
 
         switch (sector) {
             case 0:
-                r = rampup;
-                g = rampdown;
-                b = JAB_MIN_BRIGHTNESS;
+                r0 = rampup;
+                g0 = rampdown;
+                b0 = JAB_MIN_BRIGHTNESS;
                 break;
             case 1:
-                r = rampdown;
-                g = JAB_MIN_BRIGHTNESS;
-                b = rampup;
+                r0 = rampdown;
+                g0 = JAB_MIN_BRIGHTNESS;
+                b0 = rampup;
                 break;
             case 2:
-                r = JAB_MIN_BRIGHTNESS;
-                g = rampup;
-                b = rampdown;
+                r0 = JAB_MIN_BRIGHTNESS;
+                g0 = rampup;
+                b0 = rampdown;
                 break;
 
         };
 
         for (uint step = 0; step < FONT_WIDTH +
                                    INTERCHAR_SPACE; step++) {   // step though each column of the 1st char for smooth scrolling
 
-            if (beat) { //Flash on beats
-                r = JAB_MAX_BRIGHTNESS;
-                g = JAB_MAX_BRIGHTNESS;
-                b = JAB_MAX_BRIGHTNESS;
+            if (beat > 0.0f) { //Flash on beats
+                r = (uint)(beat * (float)JAB_MAX_BRIGHTNESS + (1.0f - beat) * (float)r0);
+                g = (uint)(beat * (float)JAB_MAX_BRIGHTNESS + (1.0f - beat) * (float)g0);
+                b = (uint)(beat * (float)JAB_MAX_BRIGHTNESS + (1.0f - beat) * (float)b0);
```

Comment:
I'm assuming this interpolates between MAX_BRIGHTNESS (full white) and r0,g0,b0? I imagine the desired effect we would want here is when beat 1.0 => full white, and when beat 0.0 => whatever rgb is 💃

Reply:
That's exactly what the code does. When `beat == 1.0f`, we get full white (I could add an additional `if` clause to handle this situation to save some cycles). When `beat == 0.0f`, we get the originally planned color.